hexrd/ui/interactive_template.py

```diff
@@ -27,11 +23,48 @@ def __init__(self, parent=None):
         self.translation = [0, 0]
         self.complete = False
         self.event_key = None
-        self.parent.setFocusPolicy(Qt.ClickFocus)
+        self.detector = detector
+        self.instrument = instrument
+        self._static = True
+
+        for canvas in self.image_tab_widget.active_canvases:
+            canvas.setFocusPolicy(Qt.ClickFocus)
```

Comment:
Maybe we should just put `self.setFocusPolicy(Qt.ClickFocus)` in the `ImageCanvas.__init__()` method instead?

Reply:
Sounds good